Sources/Plasma/FeatureLib/pfGameGUIMgr/pfGUIMultiLineEditCtrl.cpp

```diff
@@ -1873,6 +1873,8 @@ void pfGUIMultiLineEditCtrl::SetFontFace(const ST::string &fontFace)
     fDynTextMap->SetFont( fFontFace, fFontSize, fFontStyle,
                             HasFlag( kXparentBgnd ) ? false : true );
     fDynTextMap->CalcStringWidth( "The quick brown fox jumped over the lazy dog.", &fLineHeight );
+    // The line length might have changed due to the new font face, so re-lick the line wrapping.
```

Comment:
Code looks good to me, although I'm a little concerned that anyone is "re-licking" the line wrapping here or on line number 1888. 🤨

Reply:
:trollface:

Reply:
What does re-lick mean?

Reply:
Apparently it's a Southern-ism for "re-do". At least, that's what Adam told me. He... may have been trolling me; I'm not 100% sure. 🤷

Reply:
https://www.urbandictionary.com/define.php?term=Relick

do I need to change it? 😛

Reply:
Hah, I guess it's pretty regional then.  Haven't heard it out here, either.  It's _fairly_ self-explanatory in context, I just wasn't certain it wasn't a typo.